### ISSUE TITLE
Items with missing subtitles: This channel is not responding.

### ISSUE THREAD

**Issue by flixajki** [pannal/Sub-Zero.bundle#196]
I think its the time out problem since i have quite big media database ( over 5, 6K video files ) or that the reading is not that fast since its located on Amazon Drive. ( mounted with acd_cli / fuse ) 

I set the logs to debug, and see the processing of the library 
```
2016-11-10 18:00:34,061 (7f598bfff700) :  INFO (missing_subtitles:40) - All subtitles exist for 'The Walking Dead S06E04, Here's Not Here'
2016-11-10 18:00:34,062 (7f598bfff700) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/11832'
2016-11-10 18:00:34,068 (7f598bfff700) :  INFO (missing_subtitles:40) - All subtitles exist for 'The Walking Dead S06E03, Thank You'
2016-11-10 18:00:34,069 (7f598bfff700) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/11831'
```

I played around with all advance settings and when resetting the plugin options I got this error:
```
  File "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins/Sub-Zero.bundle/Contents/Libraries/Shared/subzero/lib/dict.py", line 16, in __getattr__
    if name in self.Dict[self.store]:
RuntimeError: maximum recursion depth exceeded while calling a Python object
```

**Comment by pannal** [comment]
The first one is known and will be fixed.

I'll check for the second one. Thanks!

**Comment by pannal** [comment]
Fixed in develop-1.4 branch (hopefully)
